buildtest/executors/base.py

```diff
@@ -101,9 +103,7 @@ def run(self, builder):
         # Run each step defined for dry run
         for step in executor.steps:
             if getattr(executor, step, None):
-                executor.builder.logger.debug(
-                    "Running %s for executor %s" % (step, executor)
-                )
+                # executor.builder.logger.debug("Running %s for executor %s" % (step, executor))
```

Comment:
Should this be commented out?

Reply:
Ok i fixed this locally and looked at logfile and i am getting the message 

```
2020-04-07 13:19:49,169 [base.py:106 -   run() ] - [DEBUG] Running setup for executor [executor-local-runlocal]
2020-04-07 13:19:49,169 [base.py:106 -   run() ] - [DEBUG] Running run for executor [executor-local-runlocal]

```

Reply:
Cool! It might be possible to simplify that, but I'm okay with it for now :)